### PR TITLE
Removes trailing slash on resource, if present

### PR DESCRIPTION
**Pull request by bseeger**
-Added code to remove a trailing slash if one is present on the
URI of the resource that the user requested to export.

Resolves: https://jira.durspace.org/browse/FCREPO-2149

### CODE REVIEW

File: fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java [hunk 1]
```diff
@@ -108,14 +108,23 @@ public File getDescriptionDirectory() {
      * Sets the URI of the resource to import/export
      */
     public void setResource(final String resource) throws URISyntaxException {
-        this.resource = new URI(resource);
+        if (resource != null && (resource.lastIndexOf('/') == resource.length() - 1)) {
```
Comment (escowles) [inline fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java]: Is there a reason not to use `resource.endsWith("/")` instead?  That seems more readable to me.

File: fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java [hunk 2]
```diff
@@ -108,14 +108,23 @@ public File getDescriptionDirectory() {
...
+        }
     }
 
     /**
      * Sets the URI of the resource to import/export
      */
-    public void setResource(final URI resource) {
-        this.resource = resource;
+    public void setResource(final URI resource) throws URISyntaxException {
+        if (resource.toString().lastIndexOf('/') == resource.toString().length() - 1) {
```
Comment (escowles) [inline fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java]: Same question as above.

File: fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java (hunk 1 above)
Comment (awoods) [inline fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java]: And maybe have this first `setResource` call the second `setResource(URI)`, and consolidate this substring logic in just one method.

**Comment by awoods** [comment]
The code looks good. If the testing is good on your end, @escowles, please feel free to merge.

**Pull request merged**